### PR TITLE
Add documentation for grpc streaming v4.1.4

### PR DESCRIPTION
**Pull request by jonfung-dydx**
Add documentation on the buffered channel, startup flags, metrics, and some QA

### CODE REVIEW

**Review by jayy04 — APPROVED** [review]

File: pages/validators/full_node_streaming.md [hunk 1]
```diff
@@ -361,4 +376,11 @@ Q: What are the exec modes?
 	ExecModeEndBlock            = 101
 	ExecModePrepareCheckState   = 102
 ```
-</details>
\ No newline at end of file
+</details>
+<br>
+
+Q: I only want to listen to confirmed updates. I do not want to process optimistic fills.
+- A: You will want to only process messages from `MsgProposedOperations` DeliverTx stage. These updates will have exec mode execModeFinalize.
```
Comment (yang-dydx) [inline pages/validators/full_node_streaming.md:383]: `MsgProposedOperations` isn't really mentioned in the doc until here. I think it should be replaced with saying they should listen to the `StreamOrderbookFill` stream

File: pages/validators/full_node_streaming.md [hunk 2]
```diff
@@ -312,6 +316,8 @@ By protocol design, each validator has their own version of the orderbook and op
 
 ![full node streaming diagram](../../artifacts/full_node_streaming_diagram.jpg)
 
+Note that DeliverTx maps to exec mode `execModeFinalize`.
```
Comment (yang-dydx) [inline pages/validators/full_node_streaming.md:319]: This is a weird gotcha. Can we fix this on the full node stream side?

Comment (jonfung-dydx) [reply pages/validators/full_node_streaming.md:319]: It's on cosmos sdk end but will think about this later, don't want to block 4.1.4 with this

File: pages/validators/full_node_streaming.md (hunk 1 above)
Comment (jonfung-dydx) [reply pages/validators/full_node_streaming.md]: modified to remove this word

**Pull request merged**